rollup.config.js

```diff
@@ -33,7 +34,7 @@ export default (async () => [
     input: "./src/single-spa.js",
     output: [
       {
-        file: `./lib/umd/single-spa${isProduction ? ".min" : ".dev"}.js`,
+        file: `./lib/umd/single-spa${isProduction ? ".min" : ".dev"}.cjs`,
```

Comment:
Since the package.json `"type"` is now `"module"`, the umd build needs the CJS extension

Reply:
I guess the alternative that I've seen packages do, is to put another `package.json` with the contents of `{type: 'commonjs'}` so that you can maintain the `.js` extension.

Reply:
I don't mind the cjs extension. It's not super easy to do `echo '{\"type\": \"commonjs\"}' > lib/umd/package.json` in a fully platform agnostic way - the echo part sometimes creates invalid json characters, and also the file separators are sometimes tricky.